content/bn/docs/reference/glossary/secret.md

```diff
@@ -0,0 +1,20 @@
+---
+title: সিক্রেট
+id: secret
+date: ২০১৮/০৪/১২
+full_link: /docs/concepts/configuration/secret/
+short_description: >
+  সংবেদনশীল তথ্য, যেমন পাসওয়ার্ড, OAuth টোকেন এবং ssh কী(keys) গুলো সংরক্ষণ করে।
+
+aka:
+tags:
+- কোর-অবজেক্ট (core-object)
+- নিরাপত্তা (security)
```

Comment:
You don't have to localize tags

Reply:
In fact, if you translate them, things don't work.

Reply:
ok, It was my first time so, I made the mistakes. I won't do it again.